src/app/core/learning-object-module/bundling/bundling.service.ts

```diff
@@ -39,4 +50,49 @@ export class BundlingService {
       return throwError(error);
     }
   }
+
+  toggleBundle(
```

Comment:
I just checked in clark-service TOGGLE_BUNDLE_FILE is actually supposed to be in the file service but we didn't move it over in https://app.shortcut.com/clarkcan/story/32778/learningobject-file-module-route-updates-for-clark-refactor 
These are the only routes that should be in bundling service. and they map to this file in clark-service. https://github.com/Cyber4All/clark-service/blob/main/src/modules/learning-object-module/bundling/bundling.routehandler.ts
![Screenshot 2024-09-16 at 10 27 09 AM](https://github.com/user-attachments/assets/f3f05e60-8c88-49d2-bb87-2206f399b033)